frontend/src/features/admin-form/common/AdminFormLayout.tsx

```diff
@@ -19,6 +24,18 @@ export const AdminFormLayout = (): JSX.Element => {
   const { formId } = useParams()
   if (!formId) throw new Error('No formId provided')
 
+  const { data: { siteBannerContent, adminBannerContent } = {} } = useEnv()
+
+  const bannerContent = useMemo(
+    () => siteBannerContent || adminBannerContent,
```

Comment:
nit: should we use `??` instead of `||`?  even though there is no real difference in our use cases, but nullish coalesce is probably slightly more correct

Reply:
Can I clarify why is `??` slightly more correct? I think `||` is better because it will fall through to the next case if the first one is `undefined` or `''`, which i think is correct in both cases. (If we defined the first env vars but left it blank then `??` would return `''` which I would think is is unintended.)

Reply:
If the banner is a number (0), which is impossible in our case haha, then the wrong output will arise due to using a Boolean OR operator. Boolean OR works on all falsey values, whilst nullish coalescence only works on undefined or null, which is slightly more correct. Either way it's not wrong in this case, but something to keep in mind.

Reply:
Ah, I see what you mean. There is a possibility of empty string too. Yeah, let's keep it and add a comment so it won't be "refactored" by others haha

Reply:
added comment in [369db25](https://github.com/opengovsg/FormSG/pull/4610/commits/369db25ac2586413b5d778e73e185a08acced32a)